v2/account_claims.go

```diff
@@ -26,26 +26,45 @@ import (
 // NoLimit is used to indicate a limit field is unlimited in value.
 const NoLimit = -1
 
-// OperatorLimits are used to limit access by an account
-type OperatorLimits struct {
-	Subs            int64 `json:"subs,omitempty"`      // Max number of subscriptions
-	Conn            int64 `json:"conn,omitempty"`      // Max number of active connections
-	LeafNodeConn    int64 `json:"leaf,omitempty"`      // Max number of active leaf node connections
+type AccountLimits struct {
 	Imports         int64 `json:"imports,omitempty"`   // Max number of imports
 	Exports         int64 `json:"exports,omitempty"`   // Max number of exports
-	Data            int64 `json:"data,omitempty"`      // Max number of bytes
-	Payload         int64 `json:"payload,omitempty"`   // Max message payload
 	WildcardExports bool  `json:"wildcards,omitempty"` // Are wildcards allowed in exports
 }
 
+type NatsLimits struct {
+	Subs         int64 `json:"subs,omitempty"`    // Max number of subscriptions
+	Conn         int64 `json:"conn,omitempty"`    // Max number of active connections
+	LeafNodeConn int64 `json:"leaf,omitempty"`    // Max number of active leaf node connections
+	Data         int64 `json:"data,omitempty"`    // Max number of bytes
+	Payload      int64 `json:"payload,omitempty"` // Max message payload
+}
+
+type JetStreamLimits struct {
+	MemoryStorage int64 `json:"mem_storage,omitempty"`  // Max number of bytes stored in memory across all streams. (0 means disabled)
+	DiskStorage   int64 `json:"disk_storage,omitempty"` // Max number of bytes stored on disk across all streams. (0 means disabled)
```

Comment:
`0 means unlimited` maybe? otherwise seems good to me

Reply:
@ripienaar the comment is saying 0 is disabled - I would imagine -1 is unlimited.

Reply:
true - js code is a bit vague but indeed seems -1 is unlimited